client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -141,19 +142,25 @@ class EmailPlan extends React.Component {
 					this.setState( {
 						isLoadingEmailAccounts: false,
 						hasLoadedEmailAccounts: true,
+						isReloadingEmailAccounts: false,
 						emailAccounts: data?.accounts || [],
 					} );
 				},
 				() => {
 					this.setState( {
 						isLoadingEmailAccounts: false,
 						hasLoadedEmailAccounts: true,
+						isReloadingEmailAccounts: false,
```

Comment:
Do we really need that new flag? We will have to remove it when migrating to React Query. If it's just to avoid displaying placeholders when reloading the list of email accounts, then maybe we could use/pass `hasLoadedEmailAccounts`:

```javascript
<EmailPlanMailboxesList
  account={ this.getAccount() }
  domain={ domain }
  isLoadingEmails={ ! hasLoadedEmailAccounts }
  mailboxes={ this.getMailboxes() }
  onAccountChanged={ this.onAccountChanged }
/>
```